### PR TITLE
Add check for nonlocal on top level.

### PR DESCRIPTION
**Pull request by windelbouwman**

### CODE REVIEW

File: tests/snippets/global_nonlocal.py [hunk 1]
```diff
@@ -21,3 +22,37 @@ def y():
...
+
+try:
+    exec(src)
+except Exception as ex:
+    # print(ex)
+    pass
+else:
+    raise RuntimeError('Must raise')
+
+# Invalid syntax:
```
Comment (cthulahoops) [inline tests/snippets/global_nonlocal.py]: Are we ready to test invalid nonlocal in a function too?

File: tests/snippets/global_nonlocal.py [hunk 2]
```diff
@@ -21,3 +22,37 @@ def y():
...
 res = x()
 assert res == 3, str(res)
+
+# Invalid syntax:
+src = """
+b = 2
+global b
+"""
+
+try:
```
[the remark this answers is not in the document]
Comment (windelbouwman) [reply tests/snippets/global_nonlocal.py]: Yes, this is a good idea!

File: tests/snippets/global_nonlocal.py (hunk 1 above)
Comment (windelbouwman) [reply tests/snippets/global_nonlocal.py]: I think so, what is the wrong case here?

Comment (cthulahoops) [reply tests/snippets/global_nonlocal.py]: I was thinking simply:
```
def f():
    nonlocal x
    return x
```

Comment (windelbouwman) [reply tests/snippets/global_nonlocal.py]: Yes, this is checked. This will raise a SyntaxError.

**Comment by windelbouwman** [comment]
@cthulahoops I think this change is fine now?

**Pull request merged**